README.md

```diff
@@ -1,4 +1,12 @@
-# <img src="./docs/banner.png">
+# <img src="./docs/images/banner.png">
+
+[![Template](<https://img.shields.io/badge/Template-LINCC%20Frameworks%20Python%20Project%20Template-brightgreen>)](<https://lincc-ppt.readthedocs.io/en/latest/>)
+[![PyPI](<https://img.shields.io/pypi/v/gundam?color=blue&logo=pypi&logoColor=white>)](<https://pypi.org/project/gundam/>)
+[![Read the Docs](<https://img.shields.io/readthedocs/gundam>)](<https://gundam.readthedocs.io/>)
+[![Actions Status](https://github.com/camposandro/gundam/actions/workflows/testing-and-coverage.yml/badge.svg)](<https://gundam.readthedocs.io/>)
```

Comment:
"Actions", "Documentation" and "Linter" actions should link to their respective action status. I guess there's a chicken-and-egg problem here, though...

Reply:
I also think the amount of badges here was also unnecessary. I cleaned these up and fixed the links to point to @samotracio's repository!